packages/kbn-eslint-config/react.js

```diff
@@ -61,7 +61,14 @@ module.exports = {
     'react/no-will-update-set-state': 'error',
     'react/no-is-mounted': 'error',
     'react/no-multi-comp': ['error', { ignoreStateless: true }],
-    'react/no-unknown-property': 'error',
+    'react/no-unknown-property': ['error', { ignore: [
+      'css',
+      'kbn-canvas-height',
+      'kbn-canvas-page',
+      'kbn-canvas-shareable',
+      'kbn-canvas-url',
+      'kbn-canvas-width',
```

Comment:
@elastic/kibana-presentation I'm not sure if it's correct to simply ignore the `kbn-canvas-*` properties like this. They are used in a few files under `x-pack/plugins/canvas` as properties directly on `div` tags. Is this intended?

Reply:
On 2nd thought, I'll try and see if I can scope them to only apply to the canvas plugin if we think they shouldn't be present outside of that

Reply:
Updated